docs/agile-development/delivery-plan/README.md

```diff
@@ -0,0 +1,50 @@
+# Delivery Plan
+
+## Goals
+
+While Scrum does not require, and in fact discourages planning more than one sprint at a time, most of us work in enterprises where we are dependent on each other. For example: marketing, sales, and support needs to have know what is coming up so they can make plans for budget, people and resources. A delivery plan helps to ensure your teams are aligning with your organizational goals.
+
+A rough assessment of the planned project scope should be achievable within a reasonable time frame leveraging the available resources. The goal is to have a rough plan and estimate as a starting point, not to implement "Agilefall."
+
+Note that this is just a starting point to enable planning discussions. We expect the actual schedule to evolve and shift over time, and that you will update the scope and timeline as you progress.
+
+## Benefits
+
+- As you complete the assessment, you can refine the scope, time frame, or ask for additional resources.
+
+- As you progress in your project/product delivery, you can highlight risks to the scope, time frame, and resources.
+
+## Approach
+
+One approach you can take to accomplish is with stickies and a spreadsheet.
+
+Step 1: Stack rank the features for EVERYTHING in the backlog; including
+
+- Functional Features
+
+- [Non-functional Features](docs/TECH-LEADS-CHECKLIST.md)
```

Comment:
Remove this line, it's a dead link and out of context

Reply:
```suggestion
```